tools/dune_rule_gen/coq_module.ml

```diff
@@ -7,6 +7,8 @@
 (* Written by: Rudi Grinberg                                            *)
 (************************************************************************)
 
+let with_timing = true
```

Comment:
@coq/build-maintainers I don't known if we want to always build with timing or if there should be some way to control it.

Reply:
I don't know either, it is easy to make it param to the script if needed.